app/controllers/likes_controller.rb

```diff
@@ -0,0 +1,14 @@
+class LikesController < ApplicationController
+  def create
+    @like = Like.new
+    @like.author = current_user
+    @like.post_id = params[:post_id]
```

Comment:
- [ ] You have done well writing the `create` action to add likes to a post 👏. However, right now, a user cannot like a post because clicking on the likes counter that you have in `shared/comments_likes` partial, it does nothing. Please ensure that when a user clicks on the likes counter, the post is liked 👍

Reply:
- [ ] Kindly work in add like issue as stated by the previous reviewer because I cannot add like in the post

> * [ ]  You have done well writing the `create` action to add likes to a post clap. However, right now, a user cannot like a post because clicking on the likes counter that you have in `shared/comments_likes` partial, it does nothing. Please ensure that when a user clicks on the likes counter, the post is liked +1